mhw_library/utils.py

```diff
@@ -0,0 +1,15 @@
+# this file contains functions useful for beginnin
+import xarray as xr
+
+def create_events_file(detrended, blobs):
```

Comment:
the name `create_events_file` does not describe well what the function is doing. Also the output is an array not a file. Maybe some name like `assign_MHW_labels` (you can come up with something else you like). Also, this function will work on any SST (or even other array with another environmental variable), not just detrended one: so you can change the name to SST_array, or even data_array .

Reply:
You can actually make SST label as a parameter , and in future use same function for looking at other extreme events.

Reply:
@CassiaCai